src/utils/cryptography.ts

```diff
@@ -27,11 +27,10 @@ export interface EncryptedMessage {
  * @see https://github.com/bcomnes/nacl-blob
  */
 export async function encryptBlob(
-  plainBlob: Blob,
+  plaintext: Uint8Array,
```

Comment:
The name and docstring still reflects `Blob`: if we need both `Blob` and `Uint8Array` interfaces, it's probably better to split out a small wrapper function? (And probably one for `string`, too, to keep the trickier encoding logic in one place, rather than relying on view code to do it?)

Reply:
Cool, this should be addressed in 21f01e4.